tests/acceptance/features/apiGraphUser/showUserEmailInResult.feature

```diff
@@ -9,7 +9,7 @@ Feature: edit/search user including email
       | displayName | Brian Murphy      |
       | email       | brian@example.com |
       | password    | 1234              |
-    And the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "true" for "graph" service
+    And the config "OCIS_USER_SEARCH_DISPLAYED_ATTRIBUTES" has been set to "mail" for "graph" service
```

Comment:
maybe we should keep at least one scenario with `OCIS_SHOW_USER_EMAIL_IN_RESULTS=true` until it is completely removed.